mozilla-release/browser/base/content/browser.js

```diff
@@ -2646,6 +2646,7 @@ function BrowserOpenTab(event) {
       openTrustedLinkIn(BROWSER_NEW_TAB_URL, where, {
         relatedToCurrent,
         resolveOnNewTabCreated: resolve,
+        private: false,
```

Comment:
Please handle this for `BROWSER_NEW_TAB_URL` in forget window.

Reply:
@tsl143 Handled.